.github/workflows/publish-bb-mac.yml

```diff
@@ -130,28 +146,50 @@ jobs:
     needs: build-check
     runs-on: ubuntu-latest
     steps:
-      - name: Download artifact (amd64-darwin)
+      - name: Download barretenberg artifact (amd64-darwin)
         uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093
         with:
           name: barretenberg-amd64-darwin.tar.gz
 
-      - name: Download artifact (arm64-darwin)
+      - name: Download barretenberg artifact (arm64-darwin)
         uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093
         with:
           name: barretenberg-arm64-darwin.tar.gz
 
-      - name: Download artifact (amd64-darwin-starknet)
+      - name: Download barretenberg artifact (amd64-darwin-starknet)
         uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093
         continue-on-error: true
         with:
           name: barretenberg-amd64-darwin-starknet.tar.gz
 
-      - name: Download artifact (arm64-darwin-starknet)
+      - name: Download barretenberg artifact (arm64-darwin-starknet)
         uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093
         continue-on-error: true
         with:
           name: barretenberg-arm64-darwin-starknet.tar.gz
 
+      - name: Download libbarretenberg artifact (amd64-darwin)
+        uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093
+        with:
+          name: libbarretenberg-amd64-darwin.tar.gz
+
+      - name: Download libbarretenberg artifact (arm64-darwin)
+        uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093
+        with:
+          name: libbarretenberg-arm64-darwin.tar.gz
+
+      - name: Download libbarretenberg artifact (amd64-darwin-starknet)
+        uses: actions/download-artifact@d3f86a106a0bac45b974a628896c90dbdf5c8093
+        continue-on-error: true
+        with:
+          name: libbarretenberg-amd64-darwin-starknet.tar.gz
```

Comment:
we do not need this for starknet variants

Reply:
[2a216e6](https://github.com/AztecProtocol/aztec-packages/pull/16023/commits/2a216e6a9f3dc58bdd7ebd82f615c41097644078)